src/main/java/Command.java

```diff
@@ -0,0 +1,20 @@
+public class Command {
+    public enum CommandType {
+        ADDTASK, EXITPROGRAM, PRINTLIST, UPDATETASKSTATUS
```

Comment:
Consider adding underscores between the words in the enum names for readability, like:
```suggestion
        ADD_TASK, EXIT_PROGRAM, PRINT_LIST, UPDATE_TASK_STATUS
```

Reply:
Enums are in Pascal Case due to java convention thus, i would change it to Pascal Case